Homestead.yaml.example

```diff
@@ -0,0 +1,15 @@
+ip: 192.168.10.30
+memory: 2048
+cpus: 1
+provider: virtualbox
+authorize: ~/.ssh/id_rsa.pub
+keys:
+    - ~/.ssh/id_rsa
+folders:
+    - { map: ~/Sites/pastebin, to: /home/vagrant/pastebin }
```

Comment:
If `/home/vagrant/Code/pastebin` is going to be the default path after `make`, it might be best to immediately reflect that here. Sorry, didn't realize this before.

On a side note: I think it's pretty weird for per-project Homestead installations to make use of a `Code` subdirectory. It makes sense for the global Homestead installation but I wouldn't use it on a per-project install.